src/main/java/org/hmhb/kml/KmlDocument.java

```diff
@@ -0,0 +1,93 @@
+package org.hmhb.kml;
```

Comment:
Yeah, that might make it easier to view.

This is Java object representation of KML, so stands on its own; might as well isolate/group them.